### ISSUE TITLE
Liquid storage

### ISSUE THREAD

**Issue by cwilkinsvi** [raoulvdberge/refinedstorage#1865]
#### What happens:
Fluid is imported via importer into the system. It will import with or without a fluid disk in it. It does not go into the fluid disk, but says it's available in the system. It is not available, and has been deleted.

More poking it has shown that any interaction with inserting via the fluid grid also deletes the fluid inserted, even if that fluid is available on an external storage. 

No fluid disk, and you can insert fluid via the grid, shows the fluid available, but it isn't there.

#### What you expected to happen:
The fluid to not be imported if there isn't a spot for it, or import it and store it properly on the fluid disk that is available. 

Fluid disks can be put into the disk drive when disk drive is in item mode, but think this is a separate thing, don't think it's a cause.

#### Steps to reproduce:
1. With or without fluid disk, have one item disk available in a disk drive. Have a disk drive connected
2. import the liquid xp from a tank. It appears in the fluid grid.
3. cannot pull out, not actually there.
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.12.2
- Forge:14.23.4.2709
- Refined Storage:1.5.34
Pack DW20 Version 2.0.0

Does this issue occur on a server? [yes/no]
Yes
#### If a (crash)log is relevant for this issue, link it here:
n/a
[pastebin/gist/etc link here]

**Comment by raoulvdberge** [comment]
Are you sure you don't have any fluid storage blocks / external storages connected?

Can you reproduce it just with RS alone (no other mods)?

**Comment by raoulvdberge** [comment]
Also try with only RS (no other mods).

**Comment by cwilkinsvi** [comment]
I found the issue. After messing around with it in an empty pack, it ran fine. I loaded the problem pack up and repeated the system I had issues with. I don't believe it's actually RS that's doing anything wrong. I had a cooking for blockheads sink on an external storage for water, and what seems to be happening is RS is treating it as a multitank and storing the fluid in it, I'm guessing because the sink shows a path to an "empty spot". It's not an actual tank though the fluid is just getting voided, so any refresh to the RS system causes the storage bus to refresh, and it no longer sees that fluid available.
